src/sentry/api/endpoints/group_tags.py

```diff
@@ -5,18 +5,34 @@
 from sentry import tagstore
 from sentry.api.base import EnvironmentMixin
 from sentry.api.bases.group import GroupEndpoint
+from sentry.api.serializers import serialize
 from sentry.models import Environment
 
 
 class GroupTagsEndpoint(GroupEndpoint, EnvironmentMixin):
     def get(self, request, group):
+
+        # optional queryparam `key` can be used to get results
+        # only for specific keys.
+        keys = [tagstore.prefix_reserved_key(k)
+                for k in request.GET.getlist('key') if k] or None
+
+        # There are 2 use-cases for this method. For the 'Tags' tab we
+        # get the top 10 values, for the tag distribution bars we get 9
+        # This should ideally just be specified by the client
+        if keys:
+            value_limit = 9
```

Comment:
Should you import the constant from sentry.tagstore.base?

Reply:
Hmm, I think that the value here and the constant in tagstore represent slightly different things. These are more like "the amount of values we want want to render this UI correctly" (and really the client should send it) and the tagstore constant is more like "Don't accidentally return a million values unless specifically requested to". 

Admittedly the fact that they are both 9 is confusing.